### PR TITLE
Adds snapping fingers emote

### PR DESCRIPTION
**Pull request by imsxz**
:cl: imsxz
add: Adds a snap emote, makes a snapping noise
soundadd: adds fingersnap.ogg
/:cl:

Adds snapping emote and sound from Paradise. I can't LIVE without my snapping emote. Also this took way too many tries for me to get working, but hey it's my first ever port.

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -275,6 +275,16 @@
 	key_third_person = "smugs"
 	message = "grins smugly."
 
+/datum/emote/living/snap
```
Comment (FlattestGuitar) [inline code/modules/mob/living/emote.dm]: This emote works for all `mob/living`s. Either move it to only humans or add hilarious variants of `message` for all the types of mobs that can snap. The variables are already there, just redefine them.

Comment (SamHPurp) [reply code/modules/mob/living/emote.dm]: I detest the emote. If anything keep it to `human` only, not for all mobs.

Comment (FlattestGuitar) [reply code/modules/mob/living/emote.dm]: "**Ian** snaps his cute paws. Awwwwww."

**Comment by imsxz** [comment]
Okay boring people, now only humans can snap.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -67,6 +67,16 @@
 	key_third_person = "shrugs"
 	message = "shrugs."
 
+/datum/emote/living/carbon/human/snap
+	key = "snap"
+	key_third_person = "snaps"
+	message = "snaps their fingers."
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/living/snap/run_emote(mob/user, params)
```
Comment (FlattestGuitar) [inline code/modules/mob/living/carbon/human/emote.dm]: The path is wrong

**Comment by imsxz** [comment]
And to think, I was gonna try porting vampires but the snap emote was difficult enough *scream

**Pull request merged**